packages/lab/src/Flow/stories/Base/Asset.tsx

```diff
@@ -39,7 +40,7 @@ const classes = {
   }),
 };
 
-export const Asset: HvFlowNodeFC<AssetData> = (props) => {
+export const Asset: HvFlowNodeFC<RFNode<AssetData>> = (props) => {
```

Comment:
Can't we keep the `HvFlowNodeFC<AssetData>` API? It'd go a long way in reducing breaking changes